exp/addons/internal/controllers/clusterresourceset_controller.go

```diff
@@ -252,6 +252,13 @@ func (r *ClusterResourceSetReconciler) ApplyClusterResourceSet(ctx context.Conte
 		return err
 	}
 
+	// Ensure that the kubernetes service of remote cluster has been created before apply ClusterResourceSet.
```

Comment:
// Ensure that the Kubernetes API Server service has been created in the remote cluster before applying the ClusterResourceSet.